packages/components/src/DescriptionList/DescriptionList.mdx

```diff
@@ -53,3 +48,12 @@ You are able to pass a react element in as the second term of the data tuple.
     ]}
   />
 </Playground>
+
+---
```

Comment:
We don't need this in here. `---` renders as an `hr` and in this case it isn't visible due to the way that docz lays out the `<Playground>` area